### PR TITLE
adds wording fixes

### PR DESCRIPTION
**Pull request by Moejoe90**

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -5,7 +5,9 @@
       {
         "messagesDir": "./intl/messages/"
       }
-    ]
+    ],
+    ["@babel/plugin-proposal-decorators", { "legacy": true }],
```
Comment (krzkaczor) [inline .babelrc]: this is solved better by @lizonp PR. if he merges his PR first i will fix it.

File: app/config/.env.example [hunk 2]
```diff
@@ -1,6 +1,6 @@
 NF_RPC_PROVIDER=https://localhost:9090/node
 NF_UNIVERSE_CONTRACT_ADDRESS="0xbe84036c11964e9743f056f4e780a99d302a77c4"
 
-NF_FEATURE_EMAIL_CHANGE_ENABLED=0
+NF_FEATURE_EMAIL_CHANGE_ENABLED=1
```
Comment (krzkaczor) [inline app/config/.env.example]: i think we dont need that change upstream. Right? Just change it locally if you want to see this feature.

File: .babelrc (hunk 1 above)
Comment (Moejoe90) [reply .babelrc]: This is from your pr, how do you recommend going forward with this?
@krzkaczor

File: app/config/.env.example (hunk 2 above)
Comment (Moejoe90) [reply app/config/.env.example]: You are correct

**Pull request merged**